config/examples/Weedo/Tina2/V2/Configuration.h

```diff
@@ -1912,7 +1912,7 @@
 
 #if ENABLED(NOZZLE_PARK_FEATURE)
   // Specify a park position as { X, Y, Z_raise }
-  #define NOZZLE_PARK_POINT { (X_MAX_POS - 10), (Y_MAX_POS - 10), 20 }
+  #define NOZZLE_PARK_POINT { (X_MAX_POS - 10), (Y_MAX_POS - 10), (Z_MAX_POS - 10) }
```

Comment:
If you’re printing near max height and cancel, this could potentially damage hardware.

Reply:
I though `G27` does `G27 P0` by default then it only goes up if not already higher. 
Please confirm

Reply:
Anyway, feel free to change to `Z_MAX_POS` if you think it's necessary

Reply:
The value is a raise, not a coordinate. Does the nozzle really need to raise all the way to the top on any pause?